Lekce8HW/Car.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Drawing;
+
+public class Car : Vehicle , IDriveable
+{
+    public string Type { get; }
+	public Car(int id, int type)
+	{
+        ID = id;
+        switch (type) 
+        {
+            case 0:
+                Type = "Protector";
+                break;
+            case 1:
+                Type = "Scout";
+                break;
+            case 2:
+                Type = "Artillery";
+                break;
+            case 3:
+                Type = "Undercover";
+                break;
+            case 4:
+                Type = "Peacekeeper";
+                break;
+            default:
+                throw new ArgumentException("Invalid car type");
+
+        }
+
+    }
+    
+    public override void Repaint(ConsoleColor color) 
+    {
+        VehicleColor = color;
+    }
+    public void Drive(int distance)
+    {
+        Mileage += distance;
+    }
+    public override void EngineOnOff() 
+    {                         
+        if (EngineStatus == false)
+        {
+            EngineStatus = !EngineStatus;
+        }
+        else
+        {
+            EngineStatus = !EngineStatus;
+        }
```

Comment:
zde ten `if` a `else` nedava smysl, at ma Engine status jakkoukoliv hodnotu, tak provadis to same
```suggestion
            EngineStatus = !EngineStatus;
```